generators/extensions/dx-asset-compute-worker-1/index.js

```diff
@@ -101,20 +101,14 @@ class DxAssetComputeWorker1 extends Generator {
       'hooks',
       // value
       {
-        'post-app-run': 'adobe-asset-compute devtool'
+        'post-app-run': 'adobe-asset-compute devtool',
+        test: 'adobe-asset-compute test-worker'
```

Comment:
nice ! should that new hook be documented in https://www.adobe.io/apis/experienceplatform/project-firefly/docs.html#!AdobeDocs/project-firefly/master/guides/app-hooks.md

Reply:
> nice ! should that new hook be documented in https://www.adobe.io/apis/experienceplatform/project-firefly/docs.html#!AdobeDocs/project-firefly/master/guides/app-hooks.md

added issue for tracking https://github.com/adobe/aio-cli-plugin-app/issues/425